src/objects/stage.ts

```diff
@@ -0,0 +1,5 @@
+export let stage: HTMLDivElement
```

Comment:
this is clever, I didn't realize this worked

Reply:
Seems to work nicely. We'll see if it causes me any headaches.